packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -32,7 +32,10 @@ const AccountMenuItems = props => {
                 onClick={handleSignOut}
                 type="button"
             >
-                <FormattedMessage id={`Sign Out`} />
+                <FormattedMessage
+                    id={'accountMenu.signOutButtonText'}
```

Comment:
Yea please do not use `text` lol that's my bad.

Reply:
Since the id format is still relatively in the baking phase, and not entirely solidified just yet, I will leave the implementation here as is. Will note these suggestions for the upcoming stories. Thank you